seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/KafkaSource.java

```diff
@@ -186,6 +189,24 @@ public void prepare(Config config) throws PrepareFailException {
                                     this.metadata.getProperties().put(key, value.unwrapped()));
         }
 
+        if (config.hasPath(MESSAGE_FORMAT_ERROR_HANDLE_WAY_OPTION.key())) {
+            MessageFormatErrorHandleWay formatErrorWayOption =
+                    ReadonlyConfig.fromConfig(config).get(MESSAGE_FORMAT_ERROR_HANDLE_WAY_OPTION);
+            switch (formatErrorWayOption) {
+                case FAIL:
+                case SKIP:
+                    this.messageFormatErrorHandleWay = formatErrorWayOption;
+                    break;
+                default:
+                    throw new PrepareFailException(
```

Comment:
throw new KafkaConnectorException, seatunnel unified exception in connectors. You can refer to https://github.com/apache/incubator-seatunnel/issues/3043 see more details.

Reply:
Thank you, here I think you can directly connect to break, because the default value is set for this parameter. May I